performance/run_benchmark.sh

```diff
@@ -21,20 +29,40 @@ CRT_PATH=$(pwd)
 DMD_DIR=$(find $DMD_PATH/.. -maxdepth 1 -type d -name "dmd-2.*")
 source $DMD_DIR/activate
 
+function get_current_branch() {
+	git_path=$1
+	cd $git_path
+	cur_branch=$(git rev-parse --abbrev-ref HEAD)
+	echo $cur_branch
+}
+
+function restore_branch() {
+	git_path=$1
+	cur_branch=$2
+	cd $git_path
+	git checkout $cur_branch
+}
+
 function sync_repos() {
 	commit_sha=$1;
 
     cd $DMD_PATH;
     commit_date=$(git show -s --format=%ci $commit_sha);
 
     git checkout $commit_sha;
-    make -f posix.mak clean &> /dev/null;
-	make -f posix.mak -C compiler/src -j8 &> /dev/null;
+    make clean &> /dev/null;
+	make -j$(nproc) &> /dev/null;
 
 	cd $PHOBOS_PATH;
     git checkout "master@{$commit_date}";
-	make -f posix.mak clean &> /dev/null;
-	make -f posix.mak -j8 &> /dev/null;
+	make clean &> /dev/null;
+	make -j$(nproc) &> /dev/null;
+
+	libphobos2_a_bytes=$(stat -Lc %s generated/linux/release/64/libphobos2.a)
+	libphobos2_so_bytes=$(stat -Lc %s generated/linux/release/64/libphobos2.so)
+
+	echo "libphobos2.a size: $libphobos2_a_bytes B / $((libphobos2_a_bytes / 1024)) KiB / $((libphobos2_a_bytes / 1024 / 1024)) MiB"
+	echo "libphobos2.so size: $libphobos2_so_bytes B / $((libphobos2_so_bytes / 1024)) KiB / $((libphobos2_so_bytes / 1024 / 1024)) MiB"
```

Comment:
Only run these commands once, there's no point in running them every time you switch branches. And also print the size of DRuntime please.